user-docs/modules/ROOT/pages/fedora-iot-bootc-pmachine-example.adoc

```diff
@@ -0,0 +1,58 @@
+= Fedora IoT Bootc Image Example with Podman Machine
+
+This example walks through building and booting a Fedora IoT bootc image in a Podman machine. 
+
+To start, create a clone of the https://gitlab.com/fedora/bootc/base-images[Fedora Bootc Base Images] Git repository. This includes the Fedora
+IoT package manifest we will use to create our Fedora IoT bootc image.
+
+Within this clone, create a `Containerfile.custom` with the following contents:
+----
+FROM localhost/fedora-bootc as builder
+RUN /usr/libexec/bootc-base-imagectl build-rootfs --manifest=fedora-iot /target-rootfs
+
+FROM scratch
+COPY --from=builder /target-rootfs/ /
+LABEL containers.bootc 1
+ENV container=oci
+STOPSIGNAL SIGRTMIN+3
+CMD ["/sbin/init"]
+----
```

Comment:
I would add a note above this to the effect of "This example is based on the Fedora bootc  documentation about building scratch images; reference the upstream docs here for the latest version/information"

https://docs.fedoraproject.org/en-US/bootc/building-from-scratch/#_using_bootc_base_imagectl_build_rootfs

I had to swap out the `builder` image to use the images in the `fedora-testing` org on Quay; my Containerfile looks like:

```Dockerfile
FROM quay.io/fedora-testing/fedora-bootc:rawhide-standard as builder
RUN /usr/libexec/bootc-base-imagectl build-rootfs --manifest=fedora-iot /target-rootfs

FROM scratch
COPY --from=builder /target-rootfs/ /
RUN bootc container lint

LABEL containers.bootc 1
LABEL ostree.bootable 1
STOPSIGNAL SIGRTMIN+3
CMD ["/sbin/init"]
```

Reply:
There should be another note about using `fedora-testing` here...

Ideally, we should be able to use `quay.io/fedora/fedora-bootc:latest`, but that image on the registry is missing the `/usr/libexec/bootc-base-imagectl` command.

Reply:
Ah I just saw your comment below, I think that answers my question